packages/thumbprint-react/components/Tooltip/index.tsx

```diff
@@ -219,7 +280,37 @@ export default function Tooltip({
                                 {/* We need to let the popper instance know when the contents of the tooltip change,
                                 so it can reposition itself.
                                 https://github.com/thumbtack/thumbprint-archive/issues/1033 */}
-                                <WhenChildrenChange do={scheduleUpdate}>{text}</WhenChildrenChange>
+                                <WhenChildrenChange do={scheduleUpdate}>
+                                    <div
+                                        onFocus={(): void => {
+                                            if (closeTimeout) {
+                                                window.clearTimeout(closeTimeout);
+                                            }
+                                        }}
+                                        onBlur={hide}
```

Comment:
Can you leave comments explaining why these are needed?

Is it not possible for that logic to live in the `<Popover>` component usage on L240?

Reply:
Done